spec/controllers/profiles_controller_spec.rb

```diff
@@ -28,6 +28,31 @@
     end
   end
 
+  context "pagination" do
+    it 'displays no profiles on index page 2' do
+      create_list(:published_profile, 23, main_topic_en: 'math')
+
+      get :index, params: { page: 1 }
+      expect(assigns(:records).count).to eq 24
```

Comment:
Why is the expectation 24 although the list created only 23 profiles?

Reply:
Ada is created on top of the test. Will rewrite that to make that more clear. Thank you